### DISCUSSION TITLE
Protecting the Journal note

### DISCUSSION THREAD

**Discussion by davidszerman** [TriliumNext/Trilium#3242] [Q&A] [answered]
Hello,

is it possible to protect the Journal tree (in a protected session) ? 
- What is going to happen when I will click on the "Today" button ? Create the note in the correct tree ? Protect it ?
- Will the edited notes also be recorded ?

Or should I avoid it ? Or should I unprotect the session before doing anything ?

I could test it but I am a little bit afraid of crashing the app by doing this...

**Comment by zadam** [comment, marked as answer]
Hi, yes, it's possible and it should work fine. Key to this is that only note title and content are encrypted, attributes are clear text and therefore can be processed even without active protected session.

New notes are created by default with the protected status of their parent, so if the whole tree is protected, then the new day notes will be protected as well. Without active protected session, day notes will still be created, but unprotected.

The only thing you need to be careful about is the "Today" script - this represents the button on the left sidebar and needs to remain unprotected, otherwise will stop working.

**Reply by davidszerman** [reply]
Hi @zadam 

Thank you very much ; I really appreciate your answer and the fact that it should work.

is there a way to manage the today script ? Moving it outside of the folder ? or something else ?
And I suppose there is the same kind of issue with the template ?

**Reply by zadam** [reply]
You can move the script outside of the Journal tree or keep it there unprotected (you can protect the whole Journal subtree - that will protect also the today button note, but then unprotect that note specifically) - protected notes can have unprotected children.

It's a good idea to keep template also unprotected (although most of its functionality should work even protected as it's mostly about unencrypted attributes).

**Comment by davidszerman** [comment]
Hello,

I have moved the scripts outside of the tree and it works like a charm !

Thanks !
